ubuntu_unused_kernels.gemspec

```diff
@@ -8,9 +8,9 @@ Gem::Specification.new do |spec|
   spec.version       = UbuntuUnusedKernels::VERSION
   spec.authors       = ["Dan Carley"]
   spec.email         = ["dan.carley@gmail.com"]
```

Comment:
The Gemspec for other projects lists a @digital.cabinet-office.gov.uk mailing list as the spec.email value; specifically vCloud Tools and a few others. Do we want to do the same here?